examples/network/tcp_server_poll.py

```diff
@@ -0,0 +1,39 @@
+
+import select, socket
+
+server = socket.socket()
+server.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+server.bind(('0.0.0.0', 8080))
```

Comment:
This isn't going to work, see the docs.

Reply:
oh,sorry.
I wouldn't have put it up, If I hadn't run the code.
Because I use it in esp8266 and esp32.
![running](https://user-images.githubusercontent.com/32978053/48040231-f5db0500-e1b2-11e8-8b96-ed8ae91c7b57.png)

Reply:
But then your example is not for "MicroPython", it's for "some very specific ports of MicroPython". It won't work on other ports, and thus is a bad example of how to write MicroPython applications. And the first port with which you should test MicroPython applications is the Unix port.

Reply:
well, maybe.
It's not "MicroPython".